### PR TITLE
Remove 3 specific tests which duplicated by generic one

### PR DESCRIPTION
**Pull request by YuriiChere**

### CODE REVIEW

File: Tests/Test.Eyes.Selenium.DotNet/Resources/ExcludedTests.txt [hunk 1]
```diff
@@ -10,6 +10,9 @@ Applitools.Selenium.Tests.TestSpecialCases(Chrome, Scroll).TestCheckRegionInAVer
 Applitools.Selenium.Tests.TestSpecialCases(Chrome, Scroll).TestCheckRegionInAVeryBigFrameAfterManualSwitchToFrame
 Applitools.Selenium.Tests.TestSpecialCases(IE, Scroll).TestCheckRegionInAVeryBigFrame
 Applitools.Selenium.Tests.TestSpecialCases(IE, Scroll).TestCheckRegionInAVeryBigFrameAfterManualSwitchToFrame
+Applitools.Selenium.Tests.TestClassicApi(Chrome, CSS).TestCheckWindow
+Applitools.Selenium.Tests.TestClassicApi(Chrome, Scroll).TestCheckWindow
+Applitools.Selenium.Tests.TestClassicApi(Chrome, VG).TestCheckWindow
```
Comment (itaibh) [inline Tests/Test.Eyes.Selenium.DotNet/Resources/ExcludedTests.txt:15]: Changes to the `ExcludeTests.txt` file are irrelevant as this file isn't in use. Actually, I'd rather delete it.

File: .gitmodules [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [submodule "sdk.coverage.tests"]
 	path = sdk.coverage.tests
 	url = https://github.com/applitools/sdk.coverage.tests.git
-	branch = DotNet_POC
+	branch = DotNet_POC_test
```
Comment (itaibh) [inline .gitmodules:4]: What's the difference between `DotNet_POC` and `DotNet_POC_test` (and isn't there simply `DotNet`?)

**Review by itaibh — CHANGES_REQUESTED** [review]

File: Tests/Test.Eyes.Selenium.DotNet/Resources/ExcludedTests.txt (hunk 1 above)
Comment (YuriiChere) [reply Tests/Test.Eyes.Selenium.DotNet/Resources/ExcludedTests.txt:15]: Do this in new PR https://github.com/applitools/Eyes.Sdk.DotNet/pull/16  . Please commit it, this PR can be closed.

File: .gitmodules (hunk 2 above)
Comment (YuriiChere) [reply .gitmodules:4]: It was when updates to sdk.coverage were in DotNet_POC_test branch. They already merged.  Please see these updates in new PR https://github.com/applitools/Eyes.Sdk.DotNet/pull/16  . Please commit it, this PR can be closed.

**Comment by itaibh** [comment]
Closed by OA request

**Pull request closed without merging**